pkg/app/src/provider/language/languageProvider.tsx

```diff
@@ -0,0 +1,40 @@
+import React, { useEffect, useState } from 'react'
+import { LanguageContext } from './modules/context'
+import axios from 'axios'
+import { useRouter } from 'next/router'
+import i18n from 'i18next'
+import { initReactI18next } from 'react-i18next'
+import Backend from 'i18next-http-backend'
+
+export function LanguageProvider({ children }) {
+	const router = useRouter()
+	const { locale, defaultLocale } = router
+	const [languageData, setLanguageData] = useState()
+	const provider = {
+		languageData,
+	}
+
+	i18n.use(Backend)
```

Comment:
@jaaydenh @DarkNebula0

for now this would work, but I assume this has to move behind the graph, e.g.:
[translation service] --> [graph] --> [client]

what do you think?

Reply:
Yes, currently we need to get the translations into the frontend, then we can look at the translatioin service.